modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -37,6 +48,7 @@ G_TYPED_KERNEL(GStereo, <GMat(GMat, GMat, const StereoOutputFormat)>, "org.openc
             case StereoOutputFormat::DEPTH_FLOAT32:
                 return left.withDepth(CV_32FC1);
             case StereoOutputFormat::DISPARITY_FIXED16_11_5:
+
```

Comment:
I believe this newline is not needed here

Reply:
Removed